config/v1/types_image.go

```diff
@@ -1,6 +1,7 @@
 package v1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+import imagev1 "github.com/openshift/api/image/v1"
```

Comment:
this import shouldn't happen.  It couples our configuration to an image API that may evolve independently.

Reply:
It does in a way depend on how the image API evolves though..if there is a change in the image API, I would assume we would want that to reflect in this field. Also, if we don't base this on the image API, then what about the field in the openshiftapiserver imagepolicyconfig? Would it be ok if that depends on the image API ? It is set based on the import mode in the image config status.